setup/helmRepository.yaml

```diff
@@ -1,3 +1,37 @@
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    run: helm-registry
+  name: helm-registry
+  namespace: hmc-system
+spec:
+  containers:
+  - image: registry:2
+    name: helm-registry
+    ports:
+    - containerPort: 5000
+  dnsPolicy: ClusterFirst
+  restartPolicy: Always
```

Comment:
Just a thought, should we add (simple) persistence by using a (e.g. hostpath) volumes to store charts across reboots?

Reply:
Good point. I added the PVC that automatically creates the volume with the default kind local-path provisioner.